src/supertux/screen_manager.cpp

```diff
@@ -330,7 +330,8 @@ ScreenManager::handle_screen_switch()
 
       // move actions to a new vector since setup() might modify it
       auto actions = std::move(m_actions);
-
+      bool quitactiontriggered = false;
```

Comment:
Can you use a variable name which is better to read? (Maybe camel-case or use _).